common/src/main/java/org/jboss/hal/testsuite/testlistener/SnapshotListener.java

```diff
@@ -17,11 +17,11 @@
 import java.io.IOException;
 import java.time.LocalTime;
 
-public class TestListener extends RunListener {
+public class SnapshotListener extends RunListener {
```

Comment:
I would use `XmlSnapshotListener` or `ServerConfigSnapshotListener`. As Snapshot generally could be easily missinterpreted as something related to UI screenshots.